shell/browser/media/media_capture_devices_dispatcher.cc

```diff
@@ -47,4 +47,20 @@ void MediaCaptureDevicesDispatcher::OnSetCapturingLinkSecured(
     blink::mojom::MediaStreamType stream_type,
     bool is_secure) {}
 
+const std::optional<blink::MediaStreamDevice>
+MediaCaptureDevicesDispatcher::GetPreferredAudioDeviceForBrowserContext(
+    content::BrowserContext* browser_context,
+    const std::vector<std::string>& eligible_audio_device_ids) const {
+  const auto& devices = GetAudioCaptureDevices();
+  return devices.empty() ? std::nullopt : std::make_optional(devices.front());
```

Comment:
Do we not need to use `eligible_audio_device_ids`, chromes impl appears to filter the `devices` vector based on these ids

Reply:
@MarshallOfSound fixed